src/cpp/core/CMakeLists.txt

```diff
@@ -355,6 +354,7 @@ if (RSTUDIO_UNIT_TESTS_ENABLED)
       ${CORE_HEADER_FILES}
    )
 
+   message("Core Boost: ${Boost_LIBRARIES}")
```

Comment:
Did you intend to leave this in? (seems likely it was just for debugging?)

Reply:
Oops! No, I did not. Thanks for catching that!